include/output.hxx

```diff
@@ -29,11 +29,10 @@ class Output;
 #ifndef __OUTPUT_H__
 #define __OUTPUT_H__
 
-#include <stdio.h>
 #include "multiostream.hxx"
 #include <iostream>
 #include <fstream>
-
+#include <boutexception.hxx>
```

Comment:
Very minor, but `<>` should be reserved for system headers, with double-quotes for our headers

Reply:
Done